README.md

```diff
@@ -4,4 +4,129 @@ A tool for managing many combinations of real and fake NPM packages within your
 
 Extracted from [ember-auto-import](https://github.com/ef4/ember-auto-import). No docs here yet, but:
  - you can see how it's used in ember-auto-import/test-scenarios
- - it's all typescript so you can follow the types to learn what's possible
\ No newline at end of file
+ - it's all typescript so you can follow the types to learn what's possible
+
+## Getting started
+
+`scenario-tester` works especially well with a monorepo. We recommend that you create a new workspace package for your scenarios e.g. : 
+
+```
+mkdir test-packages/test-scenarios
+```
+
+Next you need to create some scenarios. The best way to create a scenario is from a Project, and the best way to create a project is to have a base project on disk. For example, in EmberJS you can create a new project with `ember new base-project` and it will create a new empty project for you. We can use this as a starting point for our other scenarios.
```

Comment:
Probably want --skip-npm and --skip-git flags on that ember command

Reply:
I did that when I personally ran the command but I didn't want to make this all about ember 🤔 there is nothing tying scenario-tester to ember and I thought that was an extra detail that we shouldn't include here. Does that make sense? 

I don't feel all that strongly about this so if you (or anyone) does I'm happy to add them 👍

Reply:
it does make sense, but as is, the example instruction of `ember new base-project` is misleading / wrong, which is where I'm coming from.

Reply:
for the purpose of having a less frustrating experience and a working set of commands I would add them
OR provide a command to achieve the same without using the ember-cli